src/Pages/UserPages/StripePayment/StripePayment.tsx

```diff
@@ -1,3 +1,313 @@
-export default function StripePayment() {
-  return <div>StripePayment</div>;
+import { useState, FormEvent, useEffect } from "react";
+import { useLocation, useNavigate } from "react-router-dom";
+import { loadStripe } from "@stripe/stripe-js";
+import {
+  AddressElement,
+  CardElement,
+  Elements,
+  useElements,
+  useStripe,
+} from "@stripe/react-stripe-js";
+import {
+  Stepper,
+  Step,
+  StepLabel,
+  Box,
+  Typography,
+  Button,
+  CircularProgress,
+  Paper,
+} from "@mui/material";
+import { apiClient, PORTAL_URLS } from "../../../Api/END_POINTS";
+import { AxiosError } from "axios";
+import toast from "react-hot-toast";
+import ButtonForm from "../../../Components/SharedComponents/ButtonForm/ButtonForm";
+import img from "../../../assets/images/paymentSucess.png";
+
+const stripePromise = loadStripe(
+  "pk_test_51OTjURBQWp069pqTmqhKZHNNd3kMf9TTynJtLJQIJDOSYcGM7xz3DabzCzE7bTxvuYMY0IX96OHBjsysHEKIrwCK006Mu7mKw8"
+);
+
+export default function StripePaymentStepper() {
+  return (
+    <Elements stripe={stripePromise}>
+      <PaymentProcess />
+    </Elements>
+  );
 }
+
+const steps = ["Billing Address", "Payment Details", "Confirmation"];
+
+const PaymentProcess = () => {
+  const stripe = useStripe();
+  const elements = useElements();
+  const location = useLocation();
+  const [activeStep, setActiveStep] = useState(0);
```

Comment:
We can make it using search params to avoid returning to the first step when we refresh the page.